recipes/hammer/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "hammer" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/mpapucci/Hammer/-/archive/v{{ version }}/Hammer-v{{ version }}.tar.gz
+  sha256: 8cf6ddbed81b931887d8432e956b125bdda31f7d2a8fbbe40eaff13b1e3473ab
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+  host:
+    - python
+    - root_base
+    - root-binaries
+    - root-dependencies
+    - boost-cpp
+    - hepmc2
+    - yaml-cpp
+    - numpy =1.14
```

Comment:
You should remove this pin. It will stomp on the global numpy pinnings.

Reply:
Done :+1:

The build might fail now as I think staged-recipes is inconsistent, or at least it tried to use 1.11 when I built this locally

Reply:
Right ok. the global pinnings are higher.